imports/lib/google-map.js

```diff
@@ -2,56 +2,96 @@ import React, { PropTypes } from 'react';
 import { Random } from 'meteor/random';
 import { createContainer } from 'meteor/react-meteor-data';
 import { Meteor } from 'meteor/meteor'
+import Marker from '../ui/marker'
 
 
 class GoogleMap extends React.Component {
-  componentDidMount() {
-    GoogleMaps.load(this.props.options || {key: Meteor.settings.public.google.api_key});
-    this.timer = setInterval( () => this.createMap(), 100);
-  }
-
-  componentDidUpdate() {
-    //console.log("updating...");
-  }
-
-  createMap(){
-    if (this.props.loaded) {
-      this.name = Random.id();
-      GoogleMaps.create({
-        name: this.name,
-        element: this.container,
-        options: this.props.mapOptions(),
-      });
-      this.props.onReady(this.name);
-      clearInterval(this.timer);
-     }
-  }
-
-  componentWillUnmount() {
-    if (GoogleMaps.maps[this.name]) {
-      google.maps.event.clearInstanceListeners(GoogleMaps.maps[this.name].instance);
-      delete GoogleMaps.maps[this.name];
-    }
-  }
-
-
-  render() {
-    return (
-      <div className="map-container" ref={c => (this.container = c)}>
-        {this.props.children}
-      </div>
-    );
-  }
-};
+    
+    constructor(props) {
+        super(props);
+    }
+
+    componentDidMount() {
+        GoogleMaps.load(this.props.options || {key: Meteor.settings.public.google.api_key});
+        this.timer = setInterval( () => this.createMap(), 100);
+    }
+
+    createMap(){
+        if (this.props.loaded) {
+            this.name = Random.id();
+            GoogleMaps.create({
+                name: this.name,
+                element: this.container,
+                options: this.props.mapOptions(),
+            });
+            this.props.onReady(this.name);
+            clearInterval(this.timer);
+        }
+    }
+
+    componentWillUnmount() {
+        if (GoogleMaps.maps[this.name]) {
+            google.maps.event.clearInstanceListeners(
+                GoogleMaps.maps[this.name].instance,
```

Comment:
Weird syntax here. Shouldn't the comma either not exist, or there be a second argument?